app/_src/gateway/upgrade/lts-upgrade.md

```diff
@@ -0,0 +1,217 @@
+At a high level, the process typically involves the following steps:
+
+1. **Provisioning a same-size deployment**: You need to ensure that the new cluster, which will run the upgraded version of Kong, has the same capacity and resources as the existing cluster. This ensures that both clusters can handle the same amount of traffic and workload.
+
+2. **Setting up dual-cluster deployment**: Once the new cluster is provisioned, you can start deploying your APIs and configurations to both clusters simultaneously. The dual cluster deployment allows both the old and new clusters to coexist and process requests in parallel.
+
+3. **Data synchronization**:  During the dual cluster deployment, data synchronization is crucial to ensure that both clusters have the same data. This can involve migrating data from the old cluster to the new one or setting up a shared data storage solution to keep both clusters in sync. Import the database from the old cluster to the new cluster by using a snapshot or `pg_restore`.
+
+4. **Traffic rerouting**: As the new cluster is running alongside the old one, you can start gradually routing incoming traffic to the new cluster. This process can be done gradually or through a controlled switchover mechanism to minimize any impact on users. This can be achieved by any load balancer, like DNS, Nginx, F5, or even a Kong node with Canary plugin enabled.
+
+5. **Testing and validation**: Before performing a complete switchover to the new cluster, it is essential to thoroughly test and validate the functionality of the upgraded version. This includes testing APIs, plugins, authentication mechanisms, and other functionalities to ensure they are working as expected.
+
+6. **Complete switchover**: Once you are confident that the upgraded cluster is fully functional and stable, you can redirect all incoming traffic to the new cluster. This step completes the upgrade process and decommissions the old cluster.
+
+By following this dual cluster deployment strategy, you can achieve a smooth and zero-downtime upgrade from one LTS version of Kong to another. This approach helps ensure high availability and uninterrupted service for your users throughout the upgrade process.
+
+#### In-place upgrade
+
+Plan a suitable maintenance or downtime window during which you can perform the upgrade. During this period, the {{site.base_gateway}} will be temporarily unavailable.
+
+While an in-place upgrade deployment allows you to perform the upgrade on the same infrastructure, it does require some downtime during the actual upgrade process. 
+
+For scenarios where zero downtime is critical, you would need to consider the dual cluster deployment method mentioned earlier, but that would involve additional resources and complexities.
+
+### DB-less mode
+
+DB-less mode is special in that each independent Kong node loads a copy of Kong configuration data into memory, without persistent database storage, so failure of some nodes will not spread to others.
```

Comment:
```suggestion
DB-less mode is special in that each independent Kong node loads a copy of Kong configuration data into memory, without persistent database storage, so if some nodes fail it will not spread to others.
```